plugins/gfproxyd/transactions.go

```diff
@@ -0,0 +1,38 @@
+package gfproxyd
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+)
+
```

Comment:
If you prefer, you can separate the common code in these functions into separate function. [Here](https://github.com/gluster/glusterd2/blob/517a731e829fce24807cdb3b82d13b9809307eda/plugins/glustershd/transactions.go#L47-L53) is an example.